src/main/scala/libra/ops/base.scala

```diff
@@ -32,6 +32,16 @@ object base {
       new ConversionFactor(otherConversionFactor.value * value)
   }
 
+  object ConversionFactor {
+    implicit def inductiveAngelConversionFactor[A, From <: UnitOfMeasure[Angle], To <: UnitOfMeasure[Angle], Next <: UnitOfMeasure[Angle]](
```

Comment:
Could this part be generic on any dimension `D` instead of `Angle`?

Reply:
i tried but i didn't work (`diverging implicit expansion for type libra.ops.base.ConversionFactor`), therefor i used `Angle`. I will try one more time to fix it

Reply:
Ok, so i'm waiting for your solution and i hope that i'll be able to understand them :)